nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+
+### Forwarding Topology
+
+The forwarding topology defines a queue per endpoint and a shared set of topics to do the publishing. The Azure Service Bus forwarding feature is used to forward messages from subscriptions on the shared set of topics to the input queue of each endpoint. As forwarding cannot be used in conjunction with partitioning, NServiceBus will perform the partitioning instead using a bundle of topics. 
+
+ * `NumberOfEntitiesInBundle(int)`: The number of topics in the bundle, defaults to 2.
+ * `BundlePrefix(string)`: The prefix used in the entity name to differentiate entities from the bundle from other entities.
+ 
+## Controlling Entities
+
+The topology will create the entities that it needs based on the following settings: 
+
+ * `Queues()`: Settings for queues.
+ * `Topics()`: Settings for topics.
+ * `Subscriptions()`: Settings for subscriptions.
+
+### Queues
+
+The following settings are available to define how queues should be created.
+
+ * `MaxSizeInMegabytes(SizeInMegabytes)`: The size of the queue, in megabytes.
+ * `MaxDeliveryCount(int)`: Sets the maximum delivery count, defaults to 10.
+ * `LockDuration(TimeSpan)`: The period of time that Azure Service Bus will lock a message before trying to redeliver it, defaults to 30 seconds.
+ * `ForwardDeadLetteredMessagesTo(string)`: Forward all dead lettered message to the specified entity.
+ * `ForwardDeadLetteredMessagesTo(Func<string, bool>, string)`: Forward all dead lettered message to the specified entity if the given condition is true (allows to exclude forwarding dead letters on the error queue for example).
+ * `DefaultMessageTimeToLive(TimeSpan)`: The maximum age of a message, defaults to TimeSpan.MaxValue.
+ * `EnableDeadLetteringOnMessageExpiration(bool)`: Messages that expire will be dead lettered, defaults to false.
+ * `EnableExpress(bool)`: Enables express mode, defaults to false.
+ * `EnableExpress(Func<string, bool>, string)`: Enables express mode when the given condition is true.
+ * `AutoDeleteOnIdle(TimeSpan)`: Auto deletes the entity if it hasn't been used for the given time frame.
+ * `EnablePartitioning(bool)`: Enables entity partitioning, defaults to false.
+ * `EnableBatchedOperations(bool)`: Enables server side batched operations, defaults to true.
+ * `RequiresDuplicateDetection(bool)`: Specifies whether the queue should perform duplicate detection, defaults to false.
+ * `DuplicateDetectionHistoryTimeWindow(TimeSpan)`: The time period in which duplicate detection should occur. 
+ * `SupportOrdering(bool)`: Best effort message ordering on the queue, defaults to false.
+ * `DescriptionFactory(Func<string, ReadOnlySettings, QueueDescription>)`: A factory method that allows to create a QueueDescription object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Queues API.
+
+### Topics
+ 
```

Comment:
General comment as above, since we talk about topics, could we replace all the "entites" with "topics" here?